tonic/src/transport/server/mod.rs

```diff
@@ -725,10 +725,6 @@ impl<L> Server<L> {
 
                     trace!("connection accepted");
 
-                    poll_fn(|cx| svc.poll_ready(cx))
-                        .await
-                        .map_err(super::Error::from_source)?;
-
```

Comment:
technically speaking a user can pass in a service that requires calling poll_ready so removing this could break that? I don't think we need to actually remove this?

Reply:
Whatever service users pass in will not affect this process. This process does not process the user's service.